lib/backends/sdc/containers.js

```diff
@@ -665,13 +656,8 @@ function defaultFabricNetwork(opts, container, payload, callback) {
             return;
         }
 
-        payload.networks = [ {uuid: conf.defaultnetwork} ];
-
-        if (requireExternal) {
-            payload.networks.push({uuid: opts.config.overlay.externalPool});
-        }
+        payload.networks = [ { uuid: conf.defaultnetwork, primary: true } ];
```

Comment:
Ah, got it: not it's not possible because of https://github.com/joyent/sdc-docker/pull/87/commits/bf5c44f5fbcac91f6cb590f8aa1d3d19329fbdbb#diff-d36e9a28d5d9b3702ee9fa48d5326b0dR1959 :)